evennia/help/models.py

```diff
@@ -56,13 +56,23 @@ class HelpEntry(SharedMemoryModel):
     db_key = models.CharField(
         "help key", max_length=255, unique=True, help_text="key to search for"
     )
+
+    @lazy_property
+    def key(self):
+        return self.db_key
```

Comment:
No need for these properties to be lazy. Not sure it even works that well her - lazy properties are for handlers that initialize themselves.

Reply:
When I change this to a standard python property the database entries fail to create.
Keeping notes here, will circle back to the issue when I am done with issues with the `sethelp` command entries. Possibly related.

60 - 62 changes to:
```python
@property
    def key(self):
        return self.db_key
```
Causes evennia.web.website.tests.HelpDetailTest to fail with:
![image](https://user-images.githubusercontent.com/54369722/122096098-2c05aa00-cddc-11eb-8975-150a75cb0fa9.png)

Reply:
Note changing evennia.help.models.help_category from a lazy_property descriptor to a standard property descriptor made the evennia.web.website.tests.HelpLockedDetailTest fail.
Taking note will resolve later.

Reply:
Resolved with commit: https://github.com/evennia/evennia/pull/2429/commits/d2f410ee26458d431c6ebb575e653d5791a60d94.

Created entry restarted, created another entry restart:
![image](https://user-images.githubusercontent.com/54369722/122223791-83118a80-ce81-11eb-82f1-38e3e0db16b1.png)

Both entries show up in help list view (side note they both display properly in detail view.):
![image](https://user-images.githubusercontent.com/54369722/122223946-a2a8b300-ce81-11eb-98e0-6036aed8db46.png)

Both entries show up in the admin page:
![image](https://user-images.githubusercontent.com/54369722/122224236-e996a880-ce81-11eb-9e2c-ead85523a30c.png)

I was also able to delete them without issue:
![image](https://user-images.githubusercontent.com/54369722/122224338-0501b380-ce82-11eb-926d-b39417ebd6a9.png)